dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -67,10 +67,23 @@ endpoint.httpOptions=HTTP Options
 endpoint.httpOptions$Ref=HTTP options reference
 endpoint.httpOptions.desc=HTTP protocol options for the endpoint.
 
+endpoint.remoteIp=Remote Ip Options
+endpoint.remoteIp$Ref=Remote ip options reference
+endpoint.remoteIp.desc=Remote ip options for the endpoint.
+
 endpoint.accessLog=HTTP Access Logging
 endpoint.accessLog$Ref=HTTP access logging reference
 endpoint.accessLog.desc=HTTP access logging configuration for the endpoint.
 
+http.remoteip=Remote Ip Options
+http.remoteip.desc=Remote Ip configuration
```

Comment:
Is "Remote Ip configuration" the description for "Remote Ip Options"? If so, explain differently and explain what the remote IP configuration is.